src/main/java/com/VetApp/PurrgentCare/models/Account.java

```diff
@@ -21,12 +21,6 @@ public class Account {
     private Boolean active;
     @Setter
     private Date dateCreated;
-    @Setter
-    @OneToMany(mappedBy = "account")
-    private List<Person> accountHolders;
-    @Setter
-    @OneToMany(mappedBy = "account")
-    private List<Pet> pets;
```

Comment:
This creates a circular dependency. I missed this during our first implementation, that's my fault.